pkgs/os-specific/linux/checksec/default.nix

```diff
@@ -24,6 +30,7 @@ stdenv.mkDerivation rec {
 
   patches = [
     ./0001-attempt-to-modprobe-config-before-checking-kernel.patch
+    ./0001-Drop-env-purging.patch
```

Comment:
```suggestion
    ./0002-drop-env-purging.patch
```

There is already a 0001 patch.

Reply:
And please add a comment explaining its purpose